src/main/java/io/cdap/plugin/snowflake/sink/batch/SnowflakeSinkConfig.java

```diff
@@ -23,7 +23,6 @@
 import io.cdap.plugin.snowflake.common.BaseSnowflakeConfig;
 import io.cdap.plugin.snowflake.common.client.SnowflakeAccessor;
 import io.cdap.plugin.snowflake.common.util.SchemaHelper;
-
```

Comment:
why this change?

Reply:
Reverted f1e33c02ce1b80b5471c0770c455fd65b3246e1a